src/execution/executable_query.cpp

```diff
@@ -24,6 +25,8 @@ ExecutableQuery::ExecutableQuery(const common::ManagedPointer<planner::AbstractP
   compiler::CodeGen codegen(exec_ctx.Get());
   compiler::Compiler compiler(query_id_, &codegen, physical_plan.Get());
   auto root = compiler.Compile();
+  //  terrier::execution::ast::AstPrettyPrint::Dump(std::cout, root);
```

Comment:
Remove this

Reply:
this was kept specificallly so that people can uncomment this and print their tpl.

Reply:
oh I see ok